### DISCUSSION TITLE
Support for pgpool2-compatible postgres driver

### DISCUSSION THREAD

**Discussion by AntonOfTheWoods** [encode/broadcaster#37] [Ideas]
`asyncpg` doesn't support `pgpool2`, whereas `aiopg` seems to have enough support for using broadcaster. 

I will try and find the time to create a PR

**Comment by AntonOfTheWoods** [comment]
@tomchristie , I'm not sure I understand why this is a discussion not an issue/feature request. To be honest, I thought you had abandoned the project, so was putting it here as a discussion point for exchange with others who might want pgpool-compatible pub-sub, rather than as a request you might help with implementing! Sorry!

**Reply by lovelydinosaur** [reply]
No need to apologise. Have just started to roll out discussions across a bunch of the encode projects, and am starting to try to move towards them being the first point of engagement. (With "Issues" reserved for verified bugs and feature requests that have been discussed and okayed.)

**Reply by AntonOfTheWoods** [reply]
Perfect, thanks. So do you have an opinion/comment? :)

**Reply by lovelydinosaur** [reply]
Personally, nope. I don't have enough bandwidth *myself* to have an opinion on that right now.

**Reply by AntonOfTheWoods** [reply]
I exchanged with the `pgpool2` team and initial testing appears that the problem with `asyncpg` has been worked around, and will be available in the upcoming versions of both the 4.1 and 4.2 branches of `pgpool2`.

So we can "close" this.
